README.md

```diff
@@ -34,6 +34,12 @@ On rpm based systems (dnf, yum, etc) these may be:
 libcephfs-devel librbd-devel librados-devel
 ```
 
+NOTE: CentOS users may want to use a
+[CentOS Storage SIG](https://wiki.centos.org/SpecialInterestGroup/Storage/Ceph)
+repository to enable packages for a supported ceph version.
+Example: `yum -y install centos-release-ceph-pacific`.
```

Comment:
you might want to consider using `dnf` on CentOS 8 and newer

Reply:
Thanks! content updated